.depcheckrc.yml

```diff
@@ -24,3 +24,4 @@ ignores:
   # Ignore dependencies which plug into the NPM lifecycle
   - '@lavamoat/preinstall-always-fail'
   - 'react-native'
+  - 'twrnc'
```

Comment:
Can we add a note here that these should be removed once we figure out how to keep the `storybook-react-native` dependencies in `storybook-react-native` and not the root `package.json`